lib/settings/widgets/swipe_picker.dart

```diff
@@ -1,4 +1,8 @@
+import 'package:expandable/expandable.dart';

```

Comment:
I run into imports that Android Studio flags as unnecessary, but they tend to start off that way I try to keep commits to what they are actually changing.

I could start removing them whenever AS tells me I don't need em.

It's hard to break anything because AS also automatically adds any imports I need when I try to use something that requires a certain one.